Source/BottomView/BottomContainerView.swift

```diff
@@ -62,29 +64,35 @@ open class BottomContainerView: UIView {
 
   // MARK: Initializers
 
-  public override init(frame: CGRect) {
-    super.init(frame: frame)
+  public init(configuration: Configuration?) {
```

Comment:
I personally prefer this to be non optional, using just default parameter of `Configuration()`